CONTRIBUTING.md

```diff
@@ -62,10 +62,25 @@ license your work under the terms of the [MIT License](LICENSE).
 
 ## Requirements
 
-- node 12+
+- node 16+
 
 ## Run locally
 
+Supposedly the necessary environment variables have been set with a `.env` file. You can simply copy and paste the following in a `.env` file placed at the root of this directory:
+
+```shell
+GATSBY_ALGOLIA_APP_ID=HF9WKP9QU1
+GATSBY_ALGOLIA_SEARCH_KEY=4ef9c8513249915cc20e3b32c450abcb
+```
+
+However, to set the `GATSBY_ALGOLIA_APP_ID` and `GATSBY_ALGOLIA_SEARCH_KEY` as environment variables manually, for example if you are using Linux or macOC, please do the following:
+
+```shell
+export GATSBY_ALGOLIA_APP_ID=HF9WKP9QU1
+export GATSBY_ALGOLIA_SEARCH_KEY=4ef9c8513249915cc20e3b32c450abcb
+```
+If you are using a Windows computer you can set the above values by creating new environment variables under System. 
```

Comment:
Since you have the above paragraph, its more clear than trying to tell people how to use linux or macos

```suggestion
```